Automattic/wp-calypso
# Woo Store: Remove LabelSettings and dependencies

#### Changes proposed in this Pull Request

This removes `LabelSettings` from the Woo Store code (which has been disabled).

By so doing, it also allows us to remove `AddCardDialog`, which allows removing `PaymentMethodForm`, `CreditCardFormFields`, and `PaymentCountrySelect` as none of these are used anywhere else. `PaymentCountrySelect` is the last place to use the code from `lib/cart/actions` which will also allow us to remove the `CartStore` in the near future (see https://github.com/Automattic/wp-calypso/issues/24019). 

If there's some desire to have label settings continue to work on the Woo Store, we could put a link from there to the wp-admin version, but I'm not sure it's necessary.

Fixes https://github.com/Automattic/wp-calypso/issues/48421
Fixes https://github.com/Automattic/wp-calypso/issues/48966
Related to https://github.com/Automattic/wp-calypso/issues/49433

#### Testing instructions

- Verify that the page where `LabelSettings` is being removed from cannot be accessed. This is at Store > Settings > Shipping as described [here](https://github.com/Automattic/wp-calypso/issues/48421#issuecomment-761093250).
- Verify that none of the removed components are being used anywhere else.
